srcpkgs/usbimager/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'usbimager'
+pkgname=usbimager
+version=1.0.10
+revision=1
+build_style=gnu-makefile
+hostmakedepends="libX11-devel"
```

Comment:
It should be ` makedepends=`, not `hostmakedepends=`

Reply:
if I make this change, the package can no longer be compiled for any architecture